### PR TITLE
Add support for Ruby v2.3.1

### PR DESCRIPTION
**Pull request by ghoneycutt**

### CODE REVIEW

**Comment by ghoneycutt** [comment]
@Phil-Friderici Could you please take a look at this.

File: Gemfile [hunk 1]
```diff
@@ -1,39 +1,29 @@
-source 'https://rubygems.org'
+source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
-if ENV['PUPPET_GEM_VERSION']
-  gem 'puppet', ENV['PUPPET_GEM_VERSION'], :require => false
```
Comment (Phil-Friderici) [inline Gemfile]: After discussions we prefer this way of setting the version of Puppet.
We think it's easier to read und understand :)

File: Gemfile [hunk 2]
```diff
@@ -1,39 +1,29 @@
...
 gem 'facter', '>= 1.7.0'
 gem 'rspec-puppet'
-gem 'puppet-lint', '>= 1.0', '< 3.0' # change to '~> 2.0' once the plugins got updated
+gem 'puppet-lint', '>= 1.0', '< 3.0'
 gem 'puppet-lint-absolute_classname-check'
 gem 'puppet-lint-alias-check'
 gem 'puppet-lint-empty_string-check'
 gem 'puppet-lint-file_ensure-check'
 gem 'puppet-lint-file_source_rights-check'
-gem 'puppet-lint-fileserver-check'
```
Comment (Phil-Friderici) [inline Gemfile]: yes, get rid of this :)

**Comment by ghoneycutt** [comment]
@Phil-Friderici lost track of this one.. I reverted the change to the Gemfile regarding setting the puppet version.

**Comment by ghoneycutt** [comment]
@Phil-Friderici could you please merge this

**Comment by Phil-Friderici** [comment]
continuing the work in https://github.com/Ericsson/puppet-module-motd/pull/22

**Comment by Phil-Friderici** [comment]
Thank you ! Released in v1.0.2

**Pull request merged**